.circleci/config.yml

```diff
@@ -453,14 +453,14 @@ workflows:
       - docker-test:
           name: docker-test-arm64
           platform: arm64
-          resource_class: arm.medium
+          resource_class: arm.medium+
```

Comment:
Just making sure that this isn't a leftover from debugging test failures: Is it really necessary to increase the resource classes here?

Reply:
turns out it's not :)